mtp/server.go

```diff
@@ -31,11 +31,15 @@ type LVServer struct {
 	info     InfoPayload
 	infoLock sync.Mutex
 
-	upgrader       websocket.Upgrader
-	streamClients  map[*websocket.Conn]bool
-	streamLock     sync.Mutex
-	controlClients map[*websocket.Conn]bool
-	controlLock    sync.Mutex
+	upgrader        websocket.Upgrader
+	streamClients   map[*websocket.Conn]bool
+	streamLock      sync.Mutex
+	controlClients  map[*websocket.Conn]bool
+	controlLock     sync.Mutex
+    motionClients   map[*MJPEGResponseWriter]bool
```

Comment:
Please use tabs instead of spaces. To fix: `gofmt -w mtp/*.go`

Reply:
Whoops, sorry about that. Thought I'd setup my editor to do that automatically.

I've fixed the indentation issues and also added an entry to `.editorconfig` to enforce this automatically.